revdb/revdb.gyp

```diff
@@ -488,6 +488,10 @@
 							# Error in ../../thirdparty/libsqlite/include/qry_dat.h
 							'-Werror=return-type',
 						],
+						'libraries':
+						[
+							'-lm',
+						],
 					},
```

Comment:
This should fix revzip too - it will be suffering the same problem as revdb and the dbdrivers as it also uses C++ features that require stdc++.